assets/scripts/config/config-docs.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
         ddApplicationId: '3493b4e7-ab12-4852-8836-ba96af7bc745',
         loggingHandler: 'http',
         algoliaConfig: {
-            index: 'docs_preview',
+            index: 'docs_preview_nick',
```

Comment:
```suggestion
            index: 'docs_preview',
```
Apply this before merging